sample.env

```diff
@@ -1,2 +1,7 @@
 DB_HOST=localhost
-REDIS_HOST=localhost
\ No newline at end of file
+REDIS_HOST=localhost
+FIREBASE_PROJECT_ID=INSERT_HERE
+FIREBASE_PRIVATE_KEY=INSERT_HERE
+FIREBASE_CLIENT_EMAIL=INSERT_HERE
+FIREBASE_WEB_API_KEY=INSERT_HERE
+FIREBASE_SERVICE_ACCOUNT_ID=INSERT_HERE
```

Comment:
Do you think you can create a file in this Google Drive folder with these keys so that we can share it with the team?
https://drive.google.com/drive/u/0/folders/1YhX35Vk8I0_PECrk6I2btsFPdd8BC-ow

Reply:
sure! I'll do it!